doc/protocol/README.md

```diff
@@ -70,15 +69,16 @@ $ kuksa-client ws://127.0.0.1:8090
 TLS is currently not supported.
 
 ### KUKSA databroker gRPC API
+
 The VISS Standard is not applicable for gRPC protocols. Here is an overview what the gRPC API in KUKSA databroker is capable of:
 
-  * Read: Reading VSS datapoints
-    * Reading current or target value for actuators
-    * Reading some metadata information from VSS datapoints
-  * Write: Writing VSS datapoints
-    * Writing sensor values
-    * Writing current or target value for actuators
-    * Soon: Writing some metadata information from VSS datapoints
-  * Subscription: Subscribing VSS datapoints
-    * Subscribing sensor values
-    * Subscribing current or target value for actuators
+- Read: Reading VSS datapoints
```

Comment:
Any style guide that recommends `-`? Both are valid and gives the same result, right? (ref: https://daringfireball.net/projects/markdown/syntax#list)

Reply:
same as above :)